sdk/lusid_drive/utilities/utility_functions.py

```diff
@@ -15,21 +14,17 @@ def name_to_id(item_list, target_item):
 # a path to id function would be useful to build here...
 
 
-def get_folder_id(api_factory, folder_name):
-    folders_api = api_factory.build(lusid_drive.api.FoldersApi)
-    response = folders_api.get_root_folder()
-    folder_id = name_to_id(response, folder_name)
+def get_folder_id(api_factory, folder_name, path="/"):
+    search_api = api_factory.build(SearchApi)
+    response = search_api.search(SearchBody(with_path=path, name=folder_name))
+    folder_id = name_to_id(response, folder_name, "Folder")
 
     return folder_id
 
 
-def get_file_id(api_factory, file_name, folder_id):
-    folders_api = api_factory.build(lusid_drive.api.FoldersApi)
-    response = folders_api.get_folder_contents(folder_id)
-    file_id = name_to_id(response, file_name)
+def get_file_id(api_factory, file_name, path="/"):

```

Comment:
this looks like a breaking change? do you know where its being used?

Reply:
I've updated all of the references to it within this repo. Is this something that would be being used externally? I can create a new method instead?

Reply:
I think this was a utility function for when the APIs had less coverage. It might be used in some old scheduler demo jobs but they might have fixed SDK versions and won't be affected OR they are no longer in use - either way, should be fine.

Reply:
There is a chance other people have been using it though in other jobs...

Reply:
Is there any way I can see who's using it? Is it safer if I just create a new method?

Reply:
I've updated to specify a filter when listing contents so that it only returns the specified file/folder. This is no longer a breaking change